<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame_manual_sign</class>
 <widget class="QWidget" name="Frame_manual_sign">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>153</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入学号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb_id">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>153</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_submit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>203</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>手动签到</string>
   </property>
  </widget>
  <widget class="QLabel" name="tip_name">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
